### PR TITLE
PR-2 in response to Search Feedback

### PR DESCRIPTION
**Pull request by sbatchelder**
(1) reordered and parametized instances of "Assembly Templates" and "
(2) removed erroneous search on detail which is a pass-through field for action-detail
(3) renamed Latest Action: Details to be labeled as Latest Action: Notes

see Search Feedback Issue #67

### CODE REVIEW

**Comment by ethanandrews** [comment]
Also deployed the sidney-dev branch to staging server so we can see in action: 
https://ooi-cgrdb-staging.whoi.net

**Comment by spetillo** [comment]
In Inventory, I can't search on non-Latest Action: Notes anymore... That's why we added "Notes & Details" to search on all details initially.

File: roundabout/search/views.py [hunk 1]
```diff
@@ -300,7 +300,6 @@ def get_avail_fields():
                         dict(value="old_serial_number", text="Old Serial Number", legal_lookups='STR_LOOKUP'),
                         dict(value="location__name",      text="Location", legal_lookups='STR_LOOKUP'),
                         dict(value="build__assembly__name",  text="Build", legal_lookups='STR_LOOKUP'),
-                        dict(value="detail",       text="Notes & Details", legal_lookups='STR_LOOKUP'),
```
Comment (spetillo) [inline roundabout/search/views.py:303]: This should stay. We often want to search on the "detail" field, for all of history.

File: roundabout/search/views.py [hunk 2]
```diff
@@ -445,7 +444,6 @@ def get_avail_fields():
                         dict(value="location__name",            text="Location", legal_lookups='STR_LOOKUP'),
                         dict(value="assembly__description",  text="Description", legal_lookups='STR_LOOKUP'),
                         dict(value="build_notes",                  text="Notes", legal_lookups='STR_LOOKUP'),
-                        dict(value="detail",                      text="Detail", legal_lookups='STR_LOOKUP'),
```
Comment (spetillo) [inline roundabout/search/views.py:448]: Keep this. Call it "Notes & Details" to keep consistent with searching on "detail" as we do in Inventory.

**Comment by sbatchelder** [comment]
I don't think there's something else that was changed that would directly affect this feature. hrm.

**Comment by spetillo** [comment]
It's Friday afternoon. I need to call it a day, and I don't want to cut into your weekends any more. We can come back to this on Monday morning. I think we're close! Have a good weekend, all!

**Comment by ethanandrews** [comment]
Yes, I’ll smoke test the deployment to make sure we’re all on the same page but we’ll get this figured out Monday.


On May 8, 2020, at 6:06 PM, Stephanie Petillo <notifications@github.com> wrote:

﻿

It's Friday afternoon. I need to call it a day, and I don't want to cut into your weekends any more. We can come back to this on Monday morning. I think we're close! Have a good weekend, all!

**Comment by spetillo** [comment]
After discussion with @sbatchelder about what exactly is happening, it appears that `.details` search only returns _some_ of the historical notes. Since that is not going to work for us, we decided to remove the Notes & Details (historical action details) search option from the list for this release. This will allow us to spend more time addressing this issue.

We also decided that it would be best to give Actions their own search view that would allow users to see multiple returned historical action items per Inventory and/or Builds.

File: roundabout/search/views.py (hunk 2 above)
Comment (spetillo) [reply roundabout/search/views.py:448]: Removed again due to functionality issues & desire to take better approach... see comment below.

File: roundabout/search/views.py (hunk 1 above)
Comment (spetillo) [reply roundabout/search/views.py:303]: Removed again due to functionality issues & desire to take better approach... see comment below.

**Comment by spetillo** [comment]
@ethanandrews Hopefully one last push to Staging for final test!

**Comment by ethanandrews** [comment]
New version is now up on staging: https://ooi-cgrdb-staging.whoi.net/

**Comment by spetillo** [comment]
@ethanandrews Please put in your approval as reviewer 2, then good to merge.

**Pull request merged**